packages/kbn-securitysolution-exceptions-common/api/find_exception_list/find_exception_list.gen.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Find exception lists API endpoint
+ *   version: 2023-10-31
+ */
+
+import { z } from 'zod';
+import { ArrayFromString } from '@kbn/zod-helpers';
+
+import { NonEmptyString } from '@kbn/openapi-common/schemas/primitives.gen';
+import { ExceptionNamespaceType, ExceptionList } from '../model/exception_list_common.gen';
+
+export type FindExceptionListsFilter = z.infer<typeof FindExceptionListsFilter>;
+export const FindExceptionListsFilter = NonEmptyString;
+
+export type FindExceptionListsRequestQuery = z.infer<typeof FindExceptionListsRequestQuery>;
+export const FindExceptionListsRequestQuery = z.object({
+  /** 
+      * Filters the returned results according to the value of the specified field.
+
+Uses the `so type.field name:field` value syntax, where `so type` can be:
+
+- `exception-list`: Specify a space-aware exception list.
+- `exception-list-agnostic`: Specify an exception list that is shared across spaces.
+ 
+      */
+  filter: FindExceptionListsFilter.optional(),
+  /** 
+      * Determines whether the returned containers are Kibana associated with a Kibana space
+or available in all spaces (`agnostic` or `single`)
+ 
+      */
+  namespace_type: ArrayFromString(ExceptionNamespaceType).optional().default(['single']),
+  /**
+   * The page number to return
+   */
+  page: z.coerce.number().int().min(0).optional(),
+  /**
+   * The number of exception lists to return per page
+   */
+  per_page: z.coerce.number().int().min(0).optional(),
+  /**
+   * Determines which field is used to sort the results
+   */
+  sort_field: NonEmptyString.optional(),
+  /**
+   * Determines the sort order, which can be `desc` or `asc`
+   */
+  sort_order: z.enum(['desc', 'asc']).optional(),
+});
+export type FindExceptionListsRequestQueryInput = z.input<typeof FindExceptionListsRequestQuery>;
+
+export type FindExceptionListsResponse = z.infer<typeof FindExceptionListsResponse>;
+export const FindExceptionListsResponse = z.object({
+  data: z.array(ExceptionList),
+  page: z.number().int().min(0),
```

Comment:
`positive()` and `min()` are overlapping. OpenAPI provides only one way to set number limits which is `min` and `max` values. It's enough to use `.min(1)` for integers.

It would be a problem to specify a positive decimal number via `min()` but it's not a concern here.